packages/service-library/src/servicelib/rabbitmq.py

```diff
@@ -57,13 +67,21 @@ async def _get_connection(
 _RABBIT_QUEUE_MESSAGE_DEFAULT_TTL_S: Final[int] = 15 * _MINUTE
 
 
+def _get_namespaced_method_name(namespace: RPCNamespace, handler_name: str) -> str:
```

Comment:
I think the position is correct since this is used in this context. Will add a test test and will enforce max handler length. This will also help when calling the handler to detect that something is wrong, not only when registering when is normally detected by aipika

Reply:
Moved to a pedantic model in the utils module